bitnami/elasticsearch/templates/data/statefulset.yaml

```diff
@@ -124,9 +124,43 @@ spec:
           resources: {{- include "common.resources.preset" (dict "type" .Values.volumePermissions.resourcesPreset) | nindent 12 }}
           {{- end }}
           volumeMounts:
+            - name: empty-dir
+              mountPath: /tmp
+              subPath: tmp-dir
             - name: data
               mountPath: /bitnami/elasticsearch/data
         {{- end }}
+        - name: copy-default-plugins
+          image: {{ include "elasticsearch.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.coordinating.containerSecurityContext.enabled }}
```

Comment:
This is not a coordinating node but you are using the coordinating settings here. Same with the rest of nodes.